core/jazz_logs/config/dev-config.json

```diff
@@ -10,7 +10,7 @@
   "VALID_CATEGORIES": [
     "api",
     "function",
-    "sls-app"
+    "custom"
```

Comment:
`custom` is only a UI label. Jazz backend should always rely on `sls-app`.

Reply:
agree. `custom` is only in UI. All backend refers to it as `sls-app`.

Reply:
Please see #1210 and merge together once above is addressed.

Reply:
Reverted.